### PR TITLE
Use source() not prefixed with boost::

### PR DESCRIPTION
**Pull request by afabri**

### CODE REVIEW

**Comment by sloriot** [comment]
Fixes [1](https://cgal.geometryfactory.com/CGAL/Members/testsuite/CGAL-4.8-Ic-113/Surface_mesh_shortest_path_Examples/TestReport_lrineau_Ubuntu-Latest-NO_DEPRECATED_CODE.gz) and [2](https://cgal.geometryfactory.com/CGAL/Members/testsuite/CGAL-4.8-Ic-113/Surface_mesh_skeletonization/TestReport_lrineau_Ubuntu-Latest-NO_DEPRECATED_CODE.gz)

**Comment by sloriot** [comment]
Merged in CGAL-4.8-Ic-115

File: Point_set_processing_3/include/CGAL/mst_orient_normals.h [hunk 1]
```diff
@@ -394,8 +394,8 @@ create_riemannian_graph(
                 // Add edge
                 typename boost::graph_traits<Riemannian_graph>::edge_descriptor e;
                 bool inserted;
-                boost::tie(e, inserted) = boost::add_edge(boost::vertex(it_index, riemannian_graph),
-                                                          boost::vertex(neighbor_index, riemannian_graph),
+                boost::tie(e, inserted) = boost::add_edge(vertex(it_index, riemannian_graph),
+                                                          vertex(neighbor_index, riemannian_graph),
                                                           riemannian_graph);
```
Comment (lrineau) [inline Point_set_processing_3/include/CGAL/mst_orient_normals.h]: `boost::add_edge`?

File: Point_set_processing_3/include/CGAL/mst_orient_normals.h [hunk 2]
```diff
@@ -514,9 +514,9 @@ create_mst_graph(
             // check that bi-directed graph is useless
             CGAL_point_set_processing_assertion(predecessor[predecessor[i]] != i);
 
-            boost::add_edge(boost::vertex(predecessor[i], mst_graph),
-                            boost::vertex(i,     mst_graph),
-                            mst_graph);
+            add_edge(boost::vertex(predecessor[i], mst_graph),
+                     boost::vertex(i,     mst_graph),
+                     mst_graph);
```
Comment (lrineau) [inline Point_set_processing_3/include/CGAL/mst_orient_normals.h]: `boost::vertex`?

**Comment by sloriot** [comment]
Merged in CGAL-4.8-Ic-116

**Pull request merged**